test/e2e/service_file_test.go

```diff
@@ -87,7 +87,7 @@ func TestServiceCreateFromFile(t *testing.T) {
 	defer os.RemoveAll(tempDir)
 	assert.NilError(t, err)
 
-	test.CreateFile("foo.json", fmt.Sprintf(ServiceJSON, test.KnDefaultTestImage), tempDir, test.FileModeReadWrite)
+	test.CreateFile("foo.json", fmt.Sprintf(ServiceJSON, test.GetKnTestImage()), tempDir, test.FileModeReadWrite)
 	test.CreateFile("foo.yaml", fmt.Sprintf(ServiceYAML, test.KnDefaultTestImage), tempDir, test.FileModeReadWrite)
```

Comment:
Do you need to change `test.KnDefaultTestImage` to `GetKnTestImage()` in this line?

Reply:
I changed all occurrences,  is this line special?

Reply:
looks like thats the only left over occurrence @mvinkler

Reply:
Ah, I see. Totally missed that one. Thank you.